addons/website/data/website_demo.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <data>
```

Comment:
Why we need separate data tag here?

Reply:
There was no technical intentions, I thought rest of the data tags have similar kind of records, so instead adding into them I used another data tag. I saw same thing in accounts module. (/home/odoo/odoo/community/addons/account/demo/account_demo.xml)